dkms.8.in

```diff
@@ -939,6 +939,31 @@ Run no more than this number of jobs in parallel.
 .TP
 .B $compress_gzip_opts, $compress_xz_opts, $compress_zstd_opts
 Control how modules are compressed. By default, the highest available level of compression is used.
+.TP
+.B $post_transaction
+If the directive is set to any non null-value, the content of the directive will be executed as a command at the end of
+.B dkms autoinstall
+with the kernel version passed as a parameter. Can be used to trigger a global
+.B initrd
+rebuild or to call a script that performs some other tasks (calling kexec, triggering the restart of services not contained in a DKMS package, etc.). Some examples:
+
+Regenerate the ram disk for the target kernel:
+.B $post_transaction="dracut --quiet --kver $1"
+(Red Hat style) or
+.B $post_transaction="update-initramfs -k $1"
+(Debian style).
+
+Launch an arbitrary script:
+.B $post_transaction="/path/to/script.sh"
```

Comment:
It might be good to assume a user has many DKMS packages and each may have special actions to perform, or the distro wants to do multiple things post install. Would it be better to make this into a directory argument and everything under `/etc/dkms/post_transaction.d` for example is executed? This would also make it easier to package these post transactions with the package itself.